example-playbooks/replace_node/replace_node.yml

```diff
@@ -1,64 +1,66 @@
 ---
-- name: Add Scylla without starting
-  # new_node should be the IP of the node that comes into the cluster. The node being replaced should be in `replaced_node` variable below
-  hosts: new_node
-  become: true
-  vars:
-    full_inventory: false
-    start_scylla_service: false
-
-    #address of the node being replaced. Please make sure to remove it from the inventory when done.
-    replaced_node: 1.2.3.4
-
-    # variables for the Scylla node role
-    # Please use the same parameters as were used for the rest of the existing cluster
-
-    # Use the same parameters as for the rest of the cluster with the addition of the following:
-    scylla_yaml_params:
-      replace_address_first_boot: {{ replaced_node }}
-
+# This play will make sure that the node being replaced is down and prevent it from trying to re-join
+# the cluster by blacklisting its ip via iptables. It'll also define a temporary_seed var which will
+# later be used by the other plays.
+- name: Prepare nodes before replacement
+  hosts: scylla
+  become: false
```

Comment:
that's a little unexpected...

Reply:
Initially I was using `become: true` for all the plays, but I had problems because some tasks from the node role will fail if you try to run them with `sudo`.
So I noticed that in the node role the tasks which need to be executed with `sudo` have `become: true` already, so by setting `become: false` in the play I make sure that only the tasks which really need `sudo` will be executed with `sudo`

In this specific play (prepare nodes before replacement), the only task which needs `become: true` is the one using `iptables`